plugins/proxies/lib_cryptolib.ecllib

```diff
@@ -19,36 +19,72 @@
 
 export CryptoLib := SERVICE : plugin('cryptolib')
 
-  //Hashing
-  
+  //----------------------------------
+  // Hashing
+  //----------------------------------
   SET OF STRING SupportedHashAlgorithms() : c, pure, fold, entrypoint='clSupportedHashAlgorithms';
   DATA   Hash(CONST VARSTRING algorithm, CONST DATA inputData) : c, pure, entrypoint='clHash';
 
-  //Symmetric Encryption
-      
+  //----------------------------------
+  // Symmetric Encryption
+  //----------------------------------
   SET OF STRING SupportedSymmetricCipherAlgorithms() : c, pure, fold, entrypoint='clSupportedSymmetricCipherAlgorithms';
-  DATA SymmetricEncrypt(CONST VARSTRING algorithm, CONST VARSTRING passphrase, CONST DATA inputData) : c, pure, entrypoint='clSymmetricEncrypt';
+
+  //Module SymmetricEncryption
+  DATA SymmetricEncrypt(CONST VARSTRING algorithm, CONST VARSTRING passphrase, CONST DATA inputData)     : c, pure, entrypoint='clSymmetricEncrypt';
   DATA SymmetricDecrypt(CONST VARSTRING algorithm, CONST VARSTRING passphrase, CONST DATA encryptedData) : c, pure, entrypoint='clSymmetricDecrypt';
 
-  //Asymmetric Public Key Encryption
-  
+  //Module SymmEncryption
+  DATA SymEncrypt(CONST VARSTRING algorithm, CONST DATA passphrase, CONST DATA inputData)    : c, pure, entrypoint='clSymmEncrypt';
+  DATA SymDecrypt(CONST VARSTRING algorithm, CONST DATA passphrase, CONST DATA encryptedData): c, pure, entrypoint='clSymmDecrypt';
```

Comment:
Consistency.  DATA versions have Symm prefix above and Sym prefix here.  I know you don't have any choice because ECL doesn't support function overloading, but it is a bit strange to change the prefix because the type of a parameter has changed.